scss/_deprecated.scss

```diff
@@ -65,3 +89,156 @@
 		@include _oButtonsIconIconOnly;
 	}
 }
+
+
+@include oColorsSetColor('white-black-20', oColorsMix($color: 'white', $background: 'black', $percentage: 20));
+
+// Theme: primary
+
+@include oColorsSetUseCase(o-buttons-primary-normal,   text,              'white');
+@include oColorsSetUseCase(o-buttons-primary-normal,   border,            'teal-50');
+@include oColorsSetUseCase(o-buttons-primary-normal,   background,        'teal-50');
+@include oColorsSetUseCase(o-buttons-primary-normal,   _deprecated,       'Use `oButtonsGetColor` instead. Please contact Origami should you have any questions.');
```

Comment:
This deprecation message won't come up if someone uses a specific property of a usecase, will it?

Reply:
Will confirm 👍

Reply:
It does:

`color: oColorsGetColorFor('o-buttons-primary-normal', 'text');`
<img width="1440" alt="screen shot 2018-03-20 at 11 49 34" src="https://user-images.githubusercontent.com/10405691/37652851-cc4d1d60-2c34-11e8-9556-08ff62d87b61.png">


`@include oColorsFor('o-buttons-primary-normal');`
<img width="1440" alt="screen shot 2018-03-20 at 11 51 23" src="https://user-images.githubusercontent.com/10405691/37652935-0f73d1ce-2c35-11e8-8f71-5a47c73212a9.png">